README.md

```diff
@@ -32,19 +32,19 @@ which has fundamental issues (like no comments).
 
 ## This solution
 
-Put all of your scripts in a file called `package-scripts.js` and use `p-s` in a single `package.json` script:
+Put all of your scripts in a file called `npsfile.js` and use `nps` in a single `package.json` script:
 
 **package.json**
 
 ```json
 {
   "scripts": {
-    "start": "package-scripts"
+    "start": "npm-package-scripts"
   }
 }
 ```
 
-**package-scripts.js**
+**npsfile.js**
```

Comment:
To me, `npsfile` sounds a bit nondescript, since `file` in and of itself doesn't say anything about what function it's supposed to serve. Maybe `config` or something to that effect would be clearer?